src/custom/components/swap/TradeSummary.tsx

```diff
@@ -48,7 +48,7 @@ export default function TradeSummary({ trade, allowedSlippage }: { trade: TradeW
       <AutoColumn style={{ padding: '0 16px' }}>
         <RowBetween>
           <RowFixed>
-            <TYPE.black fontSize={14} fontWeight={400} color={theme.text2}>
+            <TYPE.black fontSize={14} fontWeight={400} color={theme.text1}>
```

Comment:
Complexity around targeting the <TYPE> by simple styled component overrides. Opting for a simple change here. If we need to do it otherwise, let me know.